src/AzureExtensionServer/Package-Can.appxmanifest

```diff
@@ -184,6 +184,43 @@
                       </LightMode>
                     </ThemeResources>
                   </Definition>
+                  <Definition Id="Azure_MyPRs" DisplayName="My Pull Requests" Description="Azure DevOps Developer Pull Requests Widget" IsCustomizable="false">

```

Comment:
DisplayName/Description needs localization (in all three)

Reply:
#252

Reply:
Per discussion, going to not fix this in this PR since every widget here needs localization. I'll make that a separate PR fix.